walkeeper/src/callmemaybe.rs

```diff
@@ -0,0 +1,293 @@
+//!
```

Comment:
Maybe it is a good idea to have top level docs mentioning what messages are sent when so the whole picture is covered